packages/ui-core/src/typography/text.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { withKnobs, color, select } from '@storybook/addon-knobs';
+import { loremIpsum } from 'lorem-ipsum';
+import { theme } from '../theme';
+import Text from './text.component';
+
+const Wrapper = styled.div`
+  width: 100%;
+`;
+
+const GOOGLE_FONTS_API =
+  'https://www.googleapis.com/webfonts/v1/webfonts/?key=AIzaSyC529qus-Wu8LEftuMjSSBOt7mKu5lMm2E';
+
+export default {
+  title: 'UI Core / Typhography',
+  decorators: [withKnobs],
+};
+
+export const TextKnobs = () => {
+  const mainFonts = [
```

Comment:
For future re-use in the stories - this part of code should be extracted to `@keen/storybook-utils` to avoid implementation of fonts fetch logic in each story.